pkg/eth/api.go

```diff
@@ -524,12 +524,18 @@ func (pea *PublicEthAPI) localGetTransactionReceipt(ctx context.Context, hash co
 		"logsBloom":         receipt.Bloom,
 	}
 
-	// Assign receipt status or post state.
-	if len(receipt.PostState) > 0 {
+	if blockNumber <= pea.B.Config.ChainConfig.ByzantiumBlock.Uint64() {
+		if receipt.GasUsed > tx.Gas() {
```

Comment:
I'm missing something. When will receipt.GasUsed ever be more than tx.Gas()? When a tx fails for whatever reason is it's receipt.GasUsed set to something above the tx.Gas() to indicate it failed? Seems odd but I'm not super familiar with this pre-byzantium behavior- wouldn't it need to track/report how much gas was actually used during the failed execution/reversion (because the tx sender still has to pay for whatever execution was performed prior to it reverting/failing)?

Reply:
Added an issue to track this https://github.com/vulcanize/ipld-eth-server/issues/102